core/src/test/java/io/confluent/rest/DosFilterTest.java

```diff
@@ -48,23 +55,142 @@ public void dosFilterEnabled_throttlesRequests() throws Exception {
     Server server = application.createServer();
     server.start();
 
-    Client client = ClientBuilder.newClient();
+    HttpGet request =
+        new HttpGet(UriBuilder.fromUri(server.getURI()).path("/foo").build());
+
+    CloseableHttpClient ephemeralClient =
+        HttpClients.custom()
+            .setConnectionManager(new BasicHttpClientConnectionManager())
+            .setKeepAliveStrategy((httpResponse, httpContext) -> -1)
+            .build();
```

Comment:
nit: Let's extract this and the persistent client creation in separate static factory methods.

Reply:
Done.